controls/radtimeline/features/grouping.md

```diff
@@ -12,105 +12,214 @@ position: 7
+				<telerik:RadTimeline.Intervals>
+					<telerik:YearInterval />
+					<telerik:MonthInterval />
+					<telerik:WeekInterval />
+					<telerik:DayInterval />
+				</telerik:RadTimeline.Intervals>
+			</telerik:RadTimeline>
+		</Grid>
+	{{endregion}}
+
+	#### __Figure 2: Result from Example 3__
+	![Rad Timeline-features-grouping-1-Group Expand Mode-Single](images/RadTimeline-features-grouping-1-GroupExpandMode-Single.jpg)
+
+* __Multiple__: A value of __Multiple__ indicates that multiple groups of timeline items can be expanded at a time. By default, all groups of timeline items are collapsed in this mode.      
+
+	#### __[XAML] Example 4: Setting the GroupExpandMode to Multiple__
+	{{region xaml-radtimeline-features-grouping-4}}
+		<!-- GroupPath is specified and GroupExpandMode is set to Multiple -->
+		<Grid>
+			<Grid.DataContext>
+				<local:RadTimelineGroupingViewModel />
+			</Grid.DataContext>
+			<telerik:RadTimeline PeriodStart="{Binding PeriodStart}"
+								PeriodEnd="{Binding PeriodEnd}"
+								VisiblePeriodStart="{Binding VisiblePeriodStart}"
+								VisiblePeriodEnd="{Binding VisiblePeriodEnd}"
+								StartPath="StartDate"
+								DurationPath="Duration"
+								GroupPath="GroupName"
+								GroupExpandMode="Multiple"
+								ItemsSource="{Binding TimelineItems}">
+				<telerik:RadTimeline.Intervals>
+					<telerik:YearInterval />
+					<telerik:MonthInterval />
+					<telerik:WeekInterval />
+					<telerik:DayInterval />
+				</telerik:RadTimeline.Intervals>
+			</telerik:RadTimeline>
+		</Grid>
+	{{endregion}}
+
+	#### __Figure 2: Result from Example 4__
+	![Rad Timeline-features-grouping-2-Group Expand Mode-Multiple](images/RadTimeline-features-grouping-2-GroupExpandMode-Multiple.jpg)
 
 ## Changing the default collapsed state of expandable groups
 
->The styles used in this section are __implicit__. In order to apply them you have to use NoXAML DLL-s. For more information check the [Implicit Styles](http://www.telerik.com/help/silverlight/styling-apperance-implicit-styles-overview.html) topic.          
+>The styles used in this section are __implicit__. In order to apply them you have to use [NoXAML]({%slug xaml-vs-noxaml%}#noxaml-assemblies) DLL-s. For more information check the [Implicit Styles]({%slug styling-apperance-implicit-styles-overview%}#setting-a-theme-using-implicit-styles) topic.
 
-By default, the groups of timeline items are collapsed when GroupExpandMode is set to either Single or Multiple. You can use the following style to change this default state:        
+By default, the groups of timeline items are collapsed when the __GroupExpandMode__ is set to either Single or Multiple. You can use the following style to change this default state:        
 
-#### __XAML__
-{{region radtimeline-features-grouping-3}}
+#### __[XAML] Example 5: Setting the IsExpanded property of the TimelineItemGroupControl through a style__
+{{region xaml-radtimeline-features-grouping-4}}

```

Comment:
Can we wrap the Style from the code snippet in a Window.Resources or UserControl.Resources tag. This way it will be more clear where the Style should be defined.

This applies for all implicit Styles below in the article.